compiler-rt/lib/builtins/CMakeLists.txt

```diff
@@ -761,6 +761,12 @@ else ()
 
   append_list_if(COMPILER_RT_HAS_STD_C11_FLAG -std=c11 BUILTIN_CFLAGS)
 
+  # Don't link the the CRT on Windows
+  if (MSVC)
+    set(CMAKE_MSVC_RUNTIME_LIBRARY "")
```

Comment:
I don't think this line is needed - an empty string isn't one of the valid choices for `CMAKE_MSVC_RUNTIME_LIBRARY`, so I think this just uses CMake's default anyway. So if we don't need to, we should probably avoid messing around with this variable.

Reply:
Empty string _is_ a valid value and stops it from inserting the default. Without this line we add /MD (to be fair, /Zl gets added after, but I think it's better to not erroneously add it)
See: https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html
> The value may also be the empty string ("") in which case no runtime library selection flag will be added explicitly by CMake.

Reply:
Oh, indeed - I did try to read the manual for this setting but I was apparently reading too sloppily. Then this makes sense I guess.

Technically, omitting all the `/MT` or `/MD` options should have the same effect as specifying `/MT`, but I guess it makes looks less conflicting at least.